src/_blog/libp2p-webrtc-browser-to-server.md

```diff
@@ -0,0 +1,258 @@
+
+#### Benefits
+
+WebTransport has many of the same features of WebRTC (fast, secure, multiplexed), but without requiring servers to implement the WebRTC stack.  Tney also get around the valid TLS certificate requirement.  After the noise handshake completes, libp2p can use raw WebTransport streams and avoid the need for double encryption.
+
+#### Limitations
+
+You might be asking yourself, why pick WebRTC over WebTransport in libp2p?  It's like WebRTC, but easier to implement and with less complexity.  WebTransport is not without it's limitations.
+
+WebTransport isn't supported [in all browsers](https://caniuse.com/webtransport). This lack of support is a big concern as it's unreasonable to expect users to _not_ use Firefox or Safari, or even older versions of Chromium-based browsers.
+
+Another issue is that browsers utilizing the WebTransport API can only connect to servers.  This severly limits the utility of the transport as browser-to-browser communication is critical to closing the loop on full interopability in libp2p.
+
+## Prior WebRTC Implementations
+
+The new WebRTC transport was built on the shoulder of giants.  These legacy transports proved that WebRTC was a viable solution for libp2p.
+
+### libp2p-webrtc-star
+
+libp2p-webrtc-star was released earlier this year.  This transport utilizes centralized STUN and TURN servers to handle signaling, and was [never intended](https://github.com/libp2p/js-libp2p/issues/385) to be a long-term solution. The repository was archived in November in favor of the new js-libp2p-webrtc tranport due to the dependance on centralized servers. 
+
+### libp2p-webrtc-direct
+libp2p-webrtc-direct utizes websockets to exchange SDPs, removing the need for the centralzed dependency in libp2p-webrtc-star.  While the centralized problem was solved, the servers need to have valid TLS certificates for websocket connectivity. The repository was archived in November.
+
+## Can I use WebRTC Now?
+
+Yes, you can use libp2p-webrtc in the [Rust](https://github.com/libp2p/rust-libp2p/tree/master/transports/webrtc) and [JavaScript](https://github.com/libp2p/js-libp2p-webrtc) implementations!  The [Go](https://github.com/libp2p/go-libp2p) implementation is close to complete.  Follow the [PR](https://github.com/libp2p/go-libp2p/pull/1655) to get notified when merged.
+
+## What's Next?
+
+WebRTC offers the capability for browsers to connect to each other 🎉.  This isn't currently possible in any of the active libp2p transports and represents a major achievement in libp2p.
+
+The [WebRTC browser-to-browser connectivity spec](https://github.com/libp2p/specs/pull/497) is currently being authored and development will soon start.  Follow the [PR](https://github.com/libp2p/specs/pull/497) for up-to-date information.
+
+## Resources and How you can help contribute
+
+If you would like to read further about WebRTC. Please see the libp2p:
+
+- [Documentation on WebRTC](https://github.com/libp2p/docs/pull/264)
```

Comment:
should be available relatively soon on the docs site (https://docs.libp2p.io/concepts/transports/webrtc/), https://github.com/libp2p/docs/pull/264